flask_resty/pagination.py

```diff
@@ -528,12 +531,27 @@ def get_filter(
             for i in range(len(column_cursors))
         )
 
-    def get_filter_clause(self, column_cursors):
-        previous_clauses = sa.and_(
-            column == value for column, _, value in column_cursors[:-1]
-        )
+    @staticmethod
+    def get_previous_clause(column_cursors):
+        # Invoking and_() without arguments is deprecated, and will be disallowed in a future release.
+        # For an empty and_() construct, use and_(True, *args).
+        previous_clauses = sa.and_()
+        for column, _, value in column_cursors:
+            if value is None:
+                previous_clauses = sa.and_(previous_clauses, column.is_(None))
+            else:
+                previous_clauses = sa.and_(previous_clauses, column == value)
+        return previous_clauses
+
+    @staticmethod
+    def _handle_nullable(column, value, is_nullable):
+        if is_nullable:
+            return sa.or_(column.is_(None), (column > value))
+        else:
+            return column > value
 
-        column, asc, value = column_cursors[-1]
+    def _prepare_current_clause(self, column, asc, value):
+        is_nullable = getattr(column.expression, "nullable", True)
```

Comment:
is this not `column.nullable`? what is `expression` here?

Reply:
expression is for `column_propety` its also works for regular fields